src/cryptoconditions/src/asn/OCTET_STRING.c

```diff
@@ -1731,6 +1733,15 @@ OCTET_STRING_free(asn_TYPE_descriptor_t *td, void *sptr, int contents_only) {
 		st->buf = 0;
 	}
 
+/* Attention !!!
+ * this is quick & dirty workaround for memory corruption bug on aarch64-linux-gnu
+ * - downside: allows memory leakage
+ * - issue description: On Raspberry Pi 4 @ 64bit linux, daemon crashes with "free(): invalid pointer" error
+ * - probable cause: misaligned memory access to nested structs containing pointers
+ * - TODO: use the latest asn1c compiler on CryptoConditions.asn, maybe generate cpp instead of c code... investigation in progress
+ */
+
+#if !defined(__aarch64__)
```

Comment:
This was a very smart workaround to let you sync the chain and test other stuff. I think the best way long-term is that I explicitly disable all known CCs on HUSH3, but leave open the possibility of zaddr CC's in the future, because we have our own custom "Hush CC" we can customize and allow it to make special taddr tx's in the future. Just like KMD notaries can make dpow tx's. 

New Hush Smart Chains can choose to enable or disable CC's (they have always defaulted to off), and when we have some kind of zCC in the future, we can push it to mainnet when it's ready 😄

Reply:
We can probably leave this workaround for now, so you can sync, and once my code that ignores CC's is done, this code should change to always define the stack, but if NAME=HUSH3 and arch=aarch64, just leak memory instead of crashing. Until we get this upstream CC bug fixed on aarch64